### PR TITLE
fix: update SyncSwap adapter #51

### PR DESCRIPTION
**Pull request by amedrontadora**

### CODE REVIEW

File: adapters/syncswap/src/sdk/config.ts [hunk 1]
```diff
@@ -3,5 +3,5 @@ export const enum CHAINS{
 }
 
 export const SUBGRAPH_URLS = {
-    [CHAINS.LINEA]: "https://api.studio.thegraph.com/query/62864/syncswap-graph-linea/v1.4.1.4"
+    [CHAINS.LINEA]: "https://gateway-arbitrum.network.thegraph.com/api/ce0ba3625ebbbd3c4b5a2af394dc8e47/subgraphs/id/3xpZFx5YNWzqemwdtRhyaTXVidKNnjY19XAWoHtvR6Lh"
```
Comment (nitish-91) [inline adapters/syncswap/src/sdk/config.ts:6]: @amedrontadora is the usage of arbitrum network intentional here. We need the liquidity on Linea. Please confirm if that will be received from this.

File: adapters/syncswap/src/index.ts [hunk 2]
```diff
@@ -1,66 +1,57 @@
...
-    });
 };
 
-getData().then(() => {
-    console.log("Done");
-});
 
+export const getUserTVLByBlock = async (blocks: BlockData) => {
+    const { blockNumber, blockTimestamp } = blocks
+    return await getSyncSwapPositionsForAddressByPoolAtBlock(blockNumber)
```
Comment (nitish-91) [inline adapters/syncswap/src/index.ts:52]: @amedrontadora can you please change the timestamp field to number in the schema of returned object.

File: adapters/syncswap/src/sdk/config.ts (hunk 1 above)
Comment (amedrontadora) [reply adapters/syncswap/src/sdk/config.ts:6]: This is the address provided by the graph service provider, which can obtain linea data.

File: adapters/syncswap/src/index.ts (hunk 2 above)
Comment (amedrontadora) [reply adapters/syncswap/src/index.ts:52]: Sure, updated

**Comment by RamonReis** [comment]
image
The `package.json` file seems to be missing

**Comment by amedrontadora** [comment]
> image The `package.json` file seems to be missing

@RamonReis Can you work on the test files? This PR only modified the file and did not add the file. It seems that the added step in the Extract folder name is empty.

**Pull request merged**